pkgs/development/compilers/sbcl/default.nix

```
@@ -173,6 +173,7 @@ stdenv.mkDerivation (self: rec {
 
   buildArgs = [
     "--prefix=$out"
+    "--dynamic-space-size=3072"
```

1. Why? What is the reason behind changing the default?
2. Make it configurable. Something like an argument `dynamicSpaceSize ? 3 * 1024`, plus something like

```suggestion
    "--dynamic-space-size=${dynamicSpaceSize}"
```

---

It depends on how many SBCL's you want to launch at the same time (and is it nicer or not to skip the extra argument), so I'd say that if we want to override a default without a clear justification for an exact number, we should make it configurable.